src/go/rpk/pkg/config/params.go

```diff
@@ -966,6 +966,7 @@ func (p *Params) Load(fs afero.Fs) (*Config, error) {
 	c.mergeRedpandaIntoRpk()          // merge from redpanda.yaml rpk section back to rpk.yaml, picks up final redpanda.yaml defaults
 	c.fixSchemePorts()                // strip any scheme, default any missing ports
 	c.parseDevOverrides()
+	c.migrateProfileNamespace() // migrate old cloud_cluster.namespace to cloud_cluster.resource_group.
```

Comment:
I think this needs to come above line 946, above promptDeleteOldRpkYaml

Reply:
Moved :+1: